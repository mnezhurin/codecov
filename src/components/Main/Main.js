import React, { Fragment } from 'react';

export const Main = ({ classNames = '', showMe = false, children }) => {
    return (
        <Fragment>
            <main className={classNames}>
                {children}
            </main>

            {
                showMe && <div className="Additional-block">It's me</div>
            }
        </Fragment>
    );
};
