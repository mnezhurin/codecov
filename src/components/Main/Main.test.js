import React from 'react';
import renderer from 'react-test-renderer';
import {Main} from './Main';

test('Link changes the class when hovered', () => {
    const component = renderer.create(
        <Main><div>Test text</div></Main>,
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
});