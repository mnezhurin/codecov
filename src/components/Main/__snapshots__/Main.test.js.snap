// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Link changes the class when hovered 1`] = `
<main
  className=""
>
  <div>
    Test text
  </div>
</main>
`;
